src/test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -279,9 +327,6 @@ public void SdkLookup_Must_Pick_The_Highest_Semantic_Version()
                 .Pass()
                 .And
                 .HaveStdErrContaining(Path.Combine(_exeSelectedMessage, "10000.0.0", _dotnetSdkDllMessageTerminator));
-
-            // Remove dummy folders from user dir
-            DeleteAvailableSdkVersions(_userSdkBaseDir, "9999.0.2");
```

Comment:
Causing a random issue? I notice there are other sdk versions added for this test as well that are not removed (see above on comment on possible colliding)